metals/src/main/scala/scala/meta/internal/metals/BuildServerConnection.scala

```diff
@@ -69,6 +70,11 @@ class BuildServerConnection private (
   // hasDebug is not yet available in BSP capabilities
   // https://github.com/build-server-protocol/build-server-protocol/pull/161
   def hasDebug: Boolean = isBloop || isSbt
+  def hasExpressionEvaluation: Boolean =
```

Comment:
Do you think there is any chance of baking this into the BSP spec so that a server could just declare this during initialize?

Reply:
This method is temporary, to manage the transition between scala-debug-adapter 1.x and scala-debug-adapter 2.x. Once Bloop `1.4.10` is released we won't need it anymore.

I will give it a better name like`usesScalaDebugAdapter2x` and add a comment.